drjt/My-List-Util/lib/My/List/Util.pm

```diff
@@ -0,0 +1,131 @@
+package My::List::Util;
+
+use 5.010;
+use strict;
+use warnings;
+
+use List::Util qw( sum shuffle );
```

Comment:
It's meant to be `My::List::Util`, not `My::Proxy::Onto::The::Real::List::Util`.